apps/cli/src/api.ts

```diff
@@ -8,35 +8,87 @@ import Agent, {
   StepResult,
   TaskInput,
 } from "forked-agent-protocol";
+import { exec } from "child_process";
+
+function execPromise(command: string) {
+  return new Promise(function (resolve, reject) {
+    exec(
+      command,
+      { cwd: path.join(__dirname, "../../../scripts") },
+      (error, stdout, stderr) => {
+        if (error) {
+          reject(error);
+          return;
+        }
+        console.log(stdout);
+        resolve(stdout.trim());
+      }
+    );
+  });
+}
 
 async function taskHandler(
   id: string,
   input: TaskInput | null
 ): Promise<StepHandler> {
+  const rootDir = path.join(process.cwd(), "../..");
```

Comment:
I don't think we should be relying on relative paths in code
If you use the cli through npx (after publishing to npm) this will break

Reply:
the reason i did this is that we are doing this only when interacting with the agent protocol (which is expected to be run locally, I might be wrong tho), since we need to make sure that both have the same workspace path (it's related to why we're passing `../../workspace` in `.env`); this means that if someone would interact with the cli through npm it would not run this command but rather the `cli.ts` file

Reply:
It's fine in the short term, but long term we do want to run the agent-protocol without having to clone the repo